_includes/lesson_footer.html

```diff
@@ -30,17 +30,17 @@
     <div class="col-md-6 help-links" align="right">
 	{% if page.source %}
 	{% if page.source == "Rmd" %}
-	<a href="{{site.github.repository_url}}/edit/{{ default_branch }}/{{page.path|replace: "_episodes", "_episodes_rmd" | replace: ".md", ".Rmd"}}">Edit on GitHub</a>
+	<a href="{{repo_url}}/edit/{{ default_branch }}/{{page.path|replace: "_episodes", "_episodes_rmd" | replace: ".md", ".Rmd"}}">Edit on GitHub</a>
 	{% endif %}
 	{% else %}
-	<a href="{{site.github.repository_url}}/edit/{{ default_branch }}/{{page.path}}">Edit on GitHub</a>
+	<a href="{{repo_url}}/edit/{{ default_branch }}/{{page.path}}">Edit on GitHub</a>
 	{% endif %}
 	/
-	<a href="{{ site.github.repository_url }}/blob/{{ site.github.source.branch }}/CONTRIBUTING.md">Contributing</a>
+	<a href="{{ repo_url }}/blob/{{ source_branch }}/CONTRIBUTING.md">Contributing</a>
```

Comment:
Why not simply `/CONTRIBUTING.md` ? This is a relative path, relative to the root of the website being rendered, so it should work regardless of the branch, repo_url, etc

Reply:
We have been pointing to the CONTRIBUTING file on GitHub instead of a Jekyll-rendered version. I think it makes sense to send possible contributors to GitHub directly, and it actually may make more sense to also point to the default branch here as well.

Reply:
> I think it makes sense to send possible contributors to GitHub directly

Why jump back and forth? I think all but the `Edit on GitHub` link should keep viewers on the rendered website. Information will be the same but presented in a better (rendered) way.

Reply:
I opened a separate issue to address this (#351)